scripts/utils.sh

```diff
@@ -41,6 +41,7 @@ function exportPaths () {
     export ERPNEXT_CONFIG_PATH=$DISTRO_PATH/configs/erpnext/initializer_config/
     export ERPNEXT_SCRIPTS_PATH=$DISTRO_PATH/binaries/erpnext/scripts/
     export KEYCLOAK_CONFIG_PATH=$DISTRO_PATH/configs/keycloak
+    export KEYCLOAK_THEME_BINARIES_PATH=$DISTRO_PATH/binaries/keycloak/theme
```

Comment:
We can make it a bit generic and not specific to themes.
```suggestion
    export KEYCLOAK_BINARIES_PATH=$DISTRO_PATH/binaries/keycloak
```

Reply:
@corneliouzbett So then how should the `docker-compose-overrides.yml` look like?

Something like this?
```
  keycloak:
    volumes:
      - "${KEYCLOAK_THEME_BINARIES_PATH:-keycloak-theme-binaries}/theme:/opt/bitnami/keycloak/providers"

volumes:
  keycloak-theme-binaries: ~
```

Not sure...

Reply:
Yes. 
That should work.